pkg/build/apiserver/buildgenerator/generator_test.go

```diff
@@ -306,6 +320,7 @@ func TestInstantiateWithImageTrigger(t *testing.T) {
 		statusTriggerIndex int // index of trigger in status that will be updated with the imagev1 id, if -1, no update expected
 		triggers           []buildv1.BuildTriggerPolicy
 		errorExpected      bool
+		oldBC              bool
```

Comment:
This is not sufficiently descriptive. I would use a name that indicates we are updating the spec:

```suggestion
		lastTriggeredIDInSpec   bool
```